deploy/olm-catalog/knative-kafka-operator/0.11.2/knative-kafka-operator.v0.11.2.clusterserviceversion.yaml

```diff
@@ -95,6 +95,8 @@ spec:
                 - name: OPERATOR_NAME
                   value: knative-kafka-operator
                 image: quay.io/openshift-knative/knative-kafka-operator:v0.11.2
+                args:
+                  - --filename=https://raw.githubusercontent.com/openshift/knative-eventing-contrib/release-v0.11.2/openshift/release/knative-eventing-kafka-contrib-v0.11.2.yaml,https://raw.githubusercontent.com/openshift-knative/knative-kafka-operator/master/deploy/resources/networkpolicies.yaml
```

Comment:
```suggestion
                  - --filename=https://raw.githubusercontent.com/openshift/knative-eventing-contrib/release-v0.11.2/openshift/release/knative-eventing-kafka-contrib-v0.11.2.yaml,https://raw.githubusercontent.com/openshift-knative/knative-kafka-operator/v0.11.2/deploy/resources/networkpolicies.yaml
```

cc @matzew

Reply:
well, master branch needs to use the YAML from master, I understand it. 
But the other file(release file) uses a released YAML.

Reply:
nope - that is a generic file - as a temporary work-around. master will be always the one we use for the second file 

I hope that serving fixes that issue soon - than this is no longer needed

Reply:
ok, cool then

Reply:
ok for versioning - thanks for the nit suggestion!